components/layouts/columns/columns-two-wide-left.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="section--bg-white">
+  <div class="columns--two-left-wide section__inner">
+
+    <div class="column--left">
```

Comment:
Just a note, I was actually thinking of this as an element. But seeing it as a modifier does actually make pretty good sense. Nice thinking.